index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Clicker+Script&family=Poppins:wght@400;500;600;700&display=swap');
+    </style>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="index.css">
+    <title>Protfolio Web</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <a class="logo" href="#">My Logo</a>
+        <img class="hamburg" src="assets/hamburg.svg" alt="hamburg">
+    </nav>
```

Comment:
- [ ] well done implementing the requirements but there are some issues you need to fix to proceed.
As in the requirements and to follow best practices, this project milestone should consist of two primary tags inside the `body` tag.
  -  first one is the `header`, header tag should contain two children. The first one is an `a` tag that consists of a logo and the second one is a `nav` tag that consists of a hamburger menu.
  - these above requirements are necessary to follow best practices, better semantics, and for a better user experience.
  - And the second main container should be a `section` tag that consist of the hero section and that is to follow best practices and convention.